src/client/AbstractClient.ts

```diff
@@ -216,6 +220,32 @@ class AbstractClient<D extends MetadataDefinition> {
     );
   }
 
+  /**
+   * patch an entity by its id
+   *
+   * @param {string|number} id the entity identifier
+   * @param {ReadonlyArray<PatchOp>} body list of patch operations
+   * @param {Record<string, unknown>} pathParameters path parameters, will be pass to the `getPathBase` method
+   */
+  patch(
```

Comment:
The idea was to create a new method to do some JSON patch (as you [explain in this comment](https://github.com/mapado/rest-client-js-sdk/pull/126#discussion_r1060081597)). 
It was different from update / "partial PUT" mainly because of it payload format. I think operations make the payload very easy to read.

I didn't really understand the goal of unitOfWork here so I just didn't use it to keep thing simple.